Makefile

```diff
@@ -175,4 +176,8 @@ install-local-python-deps:
 install-local-docs-deps: install-local-python-deps
 	pip install -r requirements/docs.txt
 
-.PHONY: all clean build pull docs livedocs build-docs lint run stop kill run-shell shell test test-image rebuild build-ci test-ci fresh-data djshell run-prod run-pocket run-pocket-prod build-prod test-cdn compile-requirements check-requirements install-local-python-deps install-local-docs-deps
+preflight:
+	${MAKE} install-local-python-deps
+	$ ./manage.py migrate
+
```

Comment:
@alexgibson Do you think it'd be useful to automatically trigger `npm install` here, too?